racksdb/web/ui/src/components/FiltersBar.vue

```diff
@@ -0,0 +1,130 @@
+<script lang="ts" setup>
+import { computed } from 'vue'
+import { XMarkIcon } from '@heroicons/vue/24/solid'
+
+const selectedRacks = defineModel<Array<string>>('selectedRacks', { required: true })
+const selectedEquipmentTypes = defineModel<Array<string>>('selectedEquipmentTypes', {
+  required: true
+})
+const selectedCategories = defineModel<Array<string>>('selectedCategories', { required: true })
+const selectedTags = defineModel<Array<string>>('selectedTags', { required: true })
+const inputEquipmentName = defineModel<string>('inputEquipmentName', { required: true })
+const activeFilters = computed(() => {
```

Comment:
Well, `activeFilters` does not gave to be a compute or a ref, you can reference the props provided by the models defined above. We can check out this point together on Thursday.

Reply:
Fixed in [007f6f9](https://github.com/rackslab/RacksDB/pull/79/commits/007f6f9973330c29e4204c9b1accaba9de81df71).